GitUp/Application/AppDelegate.m

```diff
@@ -411,6 +413,14 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
 #endif
 }
 
+- (void)_getUrl:(NSAppleEventDescriptor*)event withReplyEvent:(NSAppleEventDescriptor*)replyEvent {
+  NSString* urlString = [event paramDescriptorForKeyword:keyDirectObject].stringValue;
+  NSString* githubPrefix = @"github-mac://openRepo/";
+  if ([urlString hasPrefix:githubPrefix]) {
+    [self _cloneRepositoryFromURLString:[urlString stringByReplacingOccurrencesOfString:githubPrefix withString:@"" options:NSCaseInsensitiveSearch range:NSMakeRange(0, urlString.length)]];
```

Comment:
If this thing is supposed to be a URL, use an actual `NSURL` for it, then case-insensitive compare the scheme and host and make sure the path is not empty. Finally pass the path to `_cloneRepositoryFromURLString:`